nest/src/database.service.ts

```diff
@@ -15,20 +15,20 @@ import {versionCompare} from "./versions";
 
 export class Run {
   params: Record<string, unknown>;
-  cluster: string;
+  cluster: Record<string, unknown>;
   impl: Record<string, unknown>;
-  workload: string;
-  vars: string;
+  workload: Record<string, unknown>;
+  vars: Record<string, unknown>;
```

Comment:
Should these be `Record<string, unknown>` instead of `string`?

Reply:
Yea, probably